src/components/layout/focused-layout-shell.test.tsx

```diff
@@ -1,10 +1,20 @@
-import { it } from 'vitest'
+import { it, vi } from 'vitest'
 import { FocusedLayoutShell } from './focused-layout-shell'
 import { render, screen } from '@testing-library/react'
+import { useClerk, useSession } from '@clerk/nextjs'
 
 import { TestingProviders } from '@/tests/providers'
 
 it('renders without a user', async () => {
+    vi.mocked(useSession).mockReturnValue({
+        session: null,
+        isLoaded: true,
+        isSignedIn: false,
+    })
+    vi.mocked(useClerk).mockReturnValue({
+        signOut: vi.fn(),
+    } as unknown as ReturnType<typeof useClerk>)
+
```

Comment:
lets add these to the existing clerk mocks in `tests/unit.helpers.jsx` and then call `mockClerkSession()` here